tools/testsys/src/delete.rs

```diff
@@ -0,0 +1,29 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use futures::TryStreamExt;
+use log::{debug, info, warn};
+use model::test_manager::{DeleteEvent, TestManager};
+
+/// Delete all tests and resources from a testsys cluster.
+#[derive(Debug, Parser)]
+pub(crate) struct Delete {}
+
+impl Delete {
+    pub(crate) async fn run(self, client: TestManager) -> Result<()> {
+        let mut stream = client.delete_all().await.context("Unable to delete all")?;
+
+        while let Some(delete) = stream
+            .try_next()
+            .await
+            .context("A deletion error occured")?
+        {
+            match delete {
+                DeleteEvent::Starting(crd) => debug!("Starting delete for {}", crd.name()),
+                DeleteEvent::Deleted(crd) => info!("Delete finished for {}", crd.name()),
+                DeleteEvent::Failed(crd) => warn!("Delete failed for {}", crd.name()),
```

Comment:
I actually think these should should be regular print statements instead of logging statements. i.e. I think these are part of the normal CLI user interface.